jdaviz/configs/cubeviz/plugins/tests/test_parsers.py

```diff
@@ -77,8 +77,8 @@ def test_spectrum1d_with_fake_fixed_units(spectrum1d, cubeviz_helper):
     reg = subsets.get('Subset 1')
 
     assert len(subsets) == 1
-    assert_allclose(reg.lower.value, 6666.666666666667)
-    assert_allclose(reg.upper.value, 7333.333333333334)
+    assert_allclose(reg.lower.value, 6600.)
```

Comment:
Is this change correct? Does this mean it has been returning the wrong values?

Reply:
Yes. @camipacifici mentioned before that we want to get the bounds of the regions themselves, not the closest spectral axis values to those bounds (which is the current behavior on main).